### PR TITLE
Pulling up logic to capture state

### PR DESCRIPTION
**Pull request by aakarshg**
Logic used by uperf to ensure subsequent runs of ansible-runner
doesn't recreate jobs/pods, is pulled up.

### CODE REVIEW

**Comment by aakarshg** [comment]
WIP currently

File: playbook.yml [hunk 1]
```diff
@@ -1,6 +1,55 @@
 - hosts: localhost
   gather_facts: no
   tasks:
+    - name: Check for running uperf pods...
```
Comment (jtaleric) [inline playbook.yml]: s/uperf//g

File: playbook.yml [hunk 2]
```diff
@@ -1,6 +1,55 @@
...
+      set_fact:
+        prev_values: |
+            [
+            {% for item in contents %}
+              "{{ item }}:{{ contents[item] }}",
+            {% endfor %}
+            ]
+        curr_values: |
+            [
+            {% for item in uperf %}
```
Comment (jtaleric) [inline playbook.yml]: s/uperf/workload/g

File: playbook.yml [hunk 3]
```diff
@@ -1,6 +1,55 @@
...
+              "{{ item }}:{{ uperf[item] }}",
```
Comment (jtaleric) [inline playbook.yml]: s/uperf/workload/g

**Comment by aakarshg** [comment]
> I see this is a WIP, but lets add a document here... Additionally, for debugging, maybe throw a clause in the when to ignore this logic, if `debug: true`, thoughts?

Ooooh, I like the thought and although it maybe confusing. We'll add documentation on what advanced users can tweak about wth ripsaw and this will be an option in there.

**Comment by aakarshg** [comment]
Noticed some problems with the current approach, we should discuss this at length.

**Comment by jtaleric** [comment]
> Noticed some problems with the current approach, we should discuss this at length.

Do we want to discuss here?

**Comment by aakarshg** [comment]
> > Noticed some problems with the current approach, we should discuss this at length.
> 
> Do we want to discuss here?

I'll try to table a reply here as well as open this up for broad discussion on the wednesday meeting.

**Comment by aakarshg** [comment]
#176 is a better approach to solve the issue, closing this.

**Pull request closed without merging**